todos/todos.js

```diff
@@ -48,4 +71,7 @@ logoutButton.addEventListener('click', () => {
 deleteButton.addEventListener('click', async () => {
     // delete all todos
     // then refetch and display the updated list of todos
+    displayTodos();
```

Comment:
try switching the order of displayTodos() and deleteAllTodos() in this event listener to fix the bug where you have to click the button twice to delete all.